packages/builder/src/stores/backend/datasources.js

```diff
@@ -57,14 +57,18 @@ export function createDatasourcesStore() {
     return updateDatasource(response)
   }
 
-  const save = async (body, fetchSchema = false) => {
+  const save = async (body, { fetchSchema, tablesFilter } = {}) => {
+    if (fetchSchema == null) {
```

Comment:
When will this be null? This means that we actively marked it as null?

Reply:
it will be null/undefined if the option isn't passed in, wanted to maintain similar parity to the old way where everything was totally optional. At the minute if you were to pass `{ tablesFilter: [] }` then `fetchSchema` would be null. This is a bit of a weird scenario, but wanted to maintain parity with `fetchSchema` always being either `true` or `false`.